src/lib/firebaseBackend.js

```diff
@@ -94,21 +94,23 @@ class FirebaseBackend extends BackendInterface {
   }
 
   addNewDropSite({
-    dropSiteFacilityName,
-    dropSiteZip,
-    dropSiteAddress,
-    dropSiteCity,
-    dropSiteState,
-    dropSiteUrl,
+    dropSiteFacilityName = '',
+    dropSiteZip = '',
+    dropSiteAddress = '',
+    dropSiteCity = '',
+    dropSiteState = '',
+    dropSiteUrl = '',
   }) {
-    if (dropSiteFacilityName && dropSiteAddress && dropSiteZip) {
+    if (dropSiteFacilityName && dropSiteZip) {
       let newSiteObj = {
         dropSiteFacilityName,
         dropSiteZip,
         dropSiteAddress,
         dropSiteCity,
         dropSiteState,
         dropSiteUrl,
+        domain: this.firebase.auth().currentUser?.email.split('@')[1] || '',
+        user: this.firebase.auth().currentUser?.uid || '',
```

Comment:
Can this be tweaked to avoid calling the auth function twice?
```
const { currentUser } = this.firebase.auth();

let newSiteObj = {
  ...
  domain: currentUser?.email.split('@')[1] || '',
  user: currentUser?.uid || '',
}
```

(or however that would look)